### DISCUSSION TITLE
How to get human-readable output on zfs diff ?

### DISCUSSION THREAD

**Discussion by HiFiPhile** [openzfs/zfs#11785] [Q&A] [answered]
Hi,

I'm using OpenZFS 2.0.3 with Proxmox. Any path or file name with non-ascii characters with `zfs diff` is displayed with escape sequences.
As I understood it's an intended behavior, but how to get human-readable output ?

**Comment by rincebrain** [comment, marked as answer]
I personally use the following alias, though you probably shouldn't use it if untrusted input is part of your concerns.

alias zfilt='while read -r LINE;do echo -e "${LINE}";done'

Then you do zfs diff foo bar | zfilt
